modules/core/src/lib/layer.js

```diff
@@ -180,6 +180,11 @@ export default class Layer extends Component {
   }
 
   is64bitEnabled() {
```

Comment:
Move to end of class, after `// DEPRECATED` comment.

Reply:
OK